pkg/generated/clientset/versioned/clientset.go

```diff
@@ -19,6 +19,8 @@ limitations under the License.
 package versioned
```

Comment:
Are all the changes in the `pkg/generated` folder related to the new deps manager?

Reply:
It's looks like related to changes in k8s.io/code-generator invocation and used version as in dep it was bind to kubernetes version not fixed (v0.17.0 // indirect) right now.

Reply:
uh ok, thanks!